pos_order_exclude_attribute_combination/static/src/scss/ProductConfiguratorPopup.scss

```diff
@@ -0,0 +1,22 @@
+.no_combination {
```

Comment:
It'd be nice to move these css classes under a global one, to limit their domain.

A rule of thumb when you're developing a PoS module, is to define the css classes under the `.pos` class, which is added to the topmost parent element in the PoS UI.

```scss
.pos {
    .no_combination {
        ...
    }
    ...
}
```

It's even better if you can specify even more.
In this case, you know the class will be used in the ProductConfiguratorPopup, whose top element has the `product-configurator-popup` class. So you could write it like this:

```scss
.pos {
    .product-configurator-popup {
        .no_combination {
            ...
        }
        ...
    }
}
```